packs/actions/destructive-vengeance.json

```diff
@@ -29,12 +29,12 @@
                 "option": "destructive-vengeance",
                 "suboptions": [
                     {
-                        "label": "PF2E.SpecificRule.Champion.NegativeDamage",
-                        "value": "void"
+                        "label": "PF2E.SpecificRule.Champion.EvilDamage",
```

Comment:
Need to update this and possibly also re-en.json

Reply:
They've already been updated:
```
"EvilDamage": "Spirit Damage",
"NegativeDamage": "Void Damage",
```

Barbarian also has:
```
"NegativeDamage": "Void Damage",
"PositiveDamage": "Vitality Damage",
```

Should we migrate these, or wait for PC2?